Improve sensuctl response filtering section in reference (#2244)
sensu/sensu-docs: content/sensu-go/5.18/sensuctl/reference.md

See the [RBAC reference][22] for information about local user management with sensuctl.

## Response filtering

**COMMERCIAL FEATURE**: Access sensuctl response filters in the packaged Sensu Go distribution.
For more information, see [Get started with commercial features][30].

> Sensuctl supports response filtering for all [`list` commands][23].

Reviewer: ```suggestion
Sensuctl supports response filtering for all [commands using the `list` verb][23].
```

For information about response filtering methods and available label and field selectors, see [API response filtering][28].

### Sensuctl-specific syntax

You can use the same methods, selectors, and examples for sensuctl response filtering as for [API response filtering][28], except you'll format your requests with the `--label-selector` and `--field-selector` flags instead of cURL.

The standard sensuctl response filtering syntax is:

{{< highlight shell >}}

> sensuctl RESOURCE_TYPE list --SELECTOR_FLAG 'FILTER_STATEMENT'

Reviewer: ```suggestion
sensuctl RESOURCE_TYPE list --SELECTOR 'FILTER_STATEMENT'
```

{{< /highlight >}}

To create a sensuctl response filtering command:

- Replace `RESOURCE_TYPE` with the resource your filter is based on.

> - Replace `SELECTOR_FLAG` with either `label-selector` or `field-selector`, depending on which selector you want to use.

Reviewer: ```suggestion
- Replace `SELECTOR` with either `label-selector` or `field-selector`, depending on which selector you want to use.
```

Reviewer: Suggesting this for consistency with the dashboard filtering PR

| `notin`  | Not included in | `slack notin check.handlers`
| `&&`     | Logical AND     | `check.publish == true && slack in check.handlers`

For details about operators, see [API response filtering operators][47].

### Filter responses with label selectors

Use the `--label-selector` flag to filter responses using custom labels.

For example, to return entities with the `proxy_type` label set to `switch`:

{{< highlight shell >}}
sensuctl entity list --label-selector 'proxy_type == switch'
{{< /highlight >}}

### Filter responses with field selectors

Use the `--field-selector` flag to filter responses using specific [resource attributes][29].

For example, to return entities with the `switches` subscription:

{{< highlight shell >}}
sensuctl entity list --field-selector 'switches in entity.subscriptions'
{{< /highlight >}}

> To retrieve all events at `2` (CRITICAL) status:

Reviewer 2: This sounds weird to me. It might sound better with something like `all (CRITICAL) '2' event status` or `all events equaling status '2' (CRITICAL)`

Author: I see what you mean. I will fix it now.

> To retrieve all events at `2` (CRITICAL) status:

Author: ```suggestion
To retrieve all events that equal a status of `2` (CRITICAL):
```
